src/Nox.EntityFramework.Postgres/PostgresDatabaseProvider.cs

```diff
@@ -4,18 +4,20 @@
 using Nox.Types.EntityFramework.Abstractions;
 using Nox.Types.EntityFramework.Configurations;
 using Nox.Types.EntityFramework.EntityBuilderAdapter;
+using Nox.Types.EntityFramework.Enums;
 using Npgsql;
 
 namespace Nox.EntityFramework.Postgres;
 
 public class PostgresDatabaseProvider: NoxDatabaseConfigurator, INoxDatabaseProvider 
 {
+    public string ConnectionString { get; protected set; } = string.Empty;
```

Comment:
if this is for all providers? you can add them to the base class NoxDatabaseConfigurator

Reply:
Rather keep it on the provider, not all database provider will implement the configurator